kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/models/KmFeedback.java

```diff
@@ -0,0 +1,75 @@
+package com.applozic.mobicomkit.uiwidgets.kommunicate.models;
+
+/**
+ * model class for the feedback response
+ * @author shubham
+ * @date 25/July/2019
+ */
+public class KmFeedback {
+
+    public KmFeedback() {}
+
+    private int groupId;
+    private String comments;
+    private int rating;
+    private Object type;
+    private String created_at;
```

Comment:
Use proper naming convention

Reply:
I didn't decide these names. These need to match the JSON keys, for deserialization.